packages/ember-views/lib/mixins/action_support.js

```diff
@@ -116,6 +116,10 @@ export default Mixin.create({
       `Attempted to call .sendAction() with the action '${action}' on the destroyed object '${this}'.`,
       !this.isDestroying && !this.isDestroyed
     );
+    deprecate('Component#sendAction is deprecated. Please use closure actions instead.', false, {
```

Comment:
We should include `this` somewhere in the deprecation message (to help track down where the deprecation is coming from).